packages/mobile/src/send/SendAmountLegacy/SendAmount.test.tsx

```diff
@@ -144,7 +148,7 @@ describe('SendAmountLegacy', () => {
           dismissAfter: null,
           displayMethod: ErrorDisplayType.BANNER,
           message: i18n.t('needMoreFundsToSend', {
-            amountNeeded: '106.9852',
```

Comment:
Why do we need more now?

Reply:
Need more funds because sending now also includes the fee (it didn't before) so the funds they need are a bit higher.